src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Type figures = Type.values()[new Random().nextInt(Type.values().length)];
+        switch (figures) {
+            case SQUARE: {
+                Square square = new Square(RANDOM.nextInt(100),
+                        COLOR_SUPPLIER.getRandomColor().toString().toLowerCase());
+                return square;
+            }
+            case ISOSCELESTRAPEZOID: {
+                IsoscelesTrapezoid isoscelesTrapezoid = new IsoscelesTrapezoid(RANDOM.nextInt(100),
+                        RANDOM.nextInt(100), RANDOM.nextInt(100),

```

Comment:
As far as you use 100 as bound for each case, probably we can extract it to the constant variable?